index.html

```diff
@@ -82,24 +79,33 @@ <h2>Passphrase</h2>
           }
           return openpgp.key.readArmored(value).keys[0];
         }
-
-        //Still no idea...
-        function getPrivateKey() {
-          if (query('#private-key').className !== 'visible') {
-            return Promise.resolve();
-          }
-          const [key, passphrase] = queryAll('#private-key textarea');
-          return openpgp.decryptKey({
-            privateKey: getKey(key, 'private'),
-            passphrase: passphrase.value,
-          })
-          .then(({key}) => key)
-          ;
+        
+        function selectElementText(el, win) {
+            win = win || window;
+            var doc = win.document, sel, range;
+            if (win.getSelection && doc.createRange) {
+                sel = win.getSelection();
+                range = doc.createRange();
+                range.selectNodeContents(el);
+                sel.removeAllRanges();
+                sel.addRange(range);
+            } else if (doc.body.createTextRange) {
+                range = doc.body.createTextRange();
+                range.moveToElementText(el);
+                range.select();
+            }
         }
 
+	query('#add-public-key').addEventListener('click', () => {
+	  const element = document.createElement('div');
+	  element.innerHTML = publicKeyTemplate;
+	  query('#public-keys').appendChild(element);
+	});
+
+
```

Comment:
So the problem is that while you can add multiple public keys, it still just displays one encrypted message, which makes that feature not really ready yet.

If it will display multiple PGP messages eventually, there's also the problem of the copy button, which PGP message will that copy or will there be a copy button for each message?

Reply:
Didn't select the part that I wanted to select... I mean the function starting [here](https://github.com/Mikiya83/WebGPG/blob/Mikiya83-pull-upstream/index.html#L99).

Reply:
I replied for the global merge request but i think it's not a problem.
But if you have other problem with selection& copy, i can have a look !